src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
```

Comment:
Stylesheet with a .scss extension won't work directly in HTML. You need to compile it to .css and link the compiled stylesheet instead.